### DISCUSSION TITLE
How to read multi-rec files  efficiently?

### DISCUSSION THREAD

**Discussion by weihua04** [apache/incubator-mxnet#14558] [Q&A]
I made several rec files from different datasets, i 'd like to train the multi-rec files at the same time efficiently. I have tried "for" cycle to read each of the rec of a mini-batch , it works, but slow.

Does anyone has any good idea to load multi-rec files efficiently?

ps: i used imagerecorditer

**Comment by vdantu** [comment]
@mxnet-label-bot add [question]

**Comment by lanking520** [comment]
@weihua04 could you please share your code in here?

**Comment by piyushghai** [comment]
@anirudhacharya Can you possibly help here ?

**Comment by anirudhacharya** [comment]
@weihua04  while creating the rec file, were the labels packed with the image data?
